### ISSUE TITLE
BERT accuracy reduced after providing custom training..The answer is also not correct

### ISSUE THREAD

**Issue by shuvadibp** [huggingface/pytorch-pretrained-BERT#365]
I have trained Google BERT with a custom training.
I have included the exact question and answer along with the context from the input document in the training file and trained BERT.

With new generated checkpoints (ckpt) I am still getting the same wrong answer as obtained before training. However it is observed the probability returned is reduced this time, in nbest_predictions.json.

**Comment by thomwolf** [comment]
Can you give a simple self-contained script to reproduce your issue?
